.github/workflows/e2e-test.yml

```diff
@@ -25,7 +25,43 @@ jobs:
       - name: Installation
         run: yarn
       - name: Setup test-website project against master release
-        run: yarn test:build:v2
+        run: |
+          yarn test:build:v2
+          rm -rf node_modules
+      - name: Start test-website project
+        run: cd test-website && yarn start --no-open
+        env:
+          E2E_TEST: true
+      - name: Build test-website project
+        run: cd test-website && yarn build
+        env:
+          CI: true
+  yarn-v2:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Installation
+        run: yarn
+      - name: Setup test-website project against master release
+        run: |
+          KEEP_CONTAINER=true yarn test:build:v2
+          rm -rf node_modules
+      - name: Setup test-website project for Yarn v2
+        run: |
+          cd test-website
+          yarn set version berry
+          yarn config set pnpMode loose
+          yarn config set npmRegistryServer http://localhost:4873
+          yarn config set unsafeHttpWhitelist --json '["localhost"]'
+          yarn install
+          yarn add @mdx-js/react
```

Comment:
wonder what's the purpose of adding mdx react here?

Reply:
Some docusaurus client code imports `@mdx-js/react` on user's behalf, so user's package.json must declare this dependency otherwise yarn v2 cannot resolve it.

Reply:
wouldn't it be better to include the dependency on the init template directly? because this might affect yarn 2 users using the init template too no?

Reply:
done